gtsfm/two_view_estimator.py

```diff
@@ -86,4 +94,51 @@ def create_computation_graph(
             exact_intrinsics,
         )
 
-        return i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph
\ No newline at end of file
+        # if we have the expected data, evaluate the computed relative pose
+        if i2Ti1_expected_graph is not None:
+            error_graphs = dask.delayed(compute_metrics)(
+                i2Ri1_graph, i2Ui1_graph, i2Ti1_expected_graph
+            )
+        else:
+            error_graphs = (None, None)
+
+        return (
+            i2Ri1_graph,
+            i2Ui1_graph,
+            v_corr_idxs_graph,
+            error_graphs[0],
+            error_graphs[1],
+        )
+
+
+def compute_metrics(
```

Comment:
Thanks for the updates @ayushbaid . maybe this could be `compute_two_view_error_metrics()` is a  better name?

Reply:
Its already in the `two_view_estimator.py` file, so does the new name makes sense? What about `compute_relative_pose_metrics()`?

Reply:
sounds good